Frontend/src/test/components/ui/avatar.test.jsx

```
@@ -0,0 +1,91 @@
+import { vi } from "vitest";
+
+// Mock Radix Avatar primitives to bypass context requirements
+vi.mock("@radix-ui/react-avatar", () => ({
+  __esModule: true,
+  Root: React.forwardRef(({ className, children, ...props }, ref) => (
+    <span ref={ref} className={className} {...props}>
+      {children}
+    </span>
+  )),
+  Image: React.forwardRef(({ className, ...props }, ref) => (
+    <img ref={ref} className={className} {...props} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)